docs/specs/redirection.yml

```diff
@@ -88,6 +89,8 @@ inputs:
     renames:
       a.md: https://docs.microsoft.com/en-us/b?query#fragment
 outputs:
+  .publish.json: |
```

Comment:
also add a case that `redirect_document_id == false`?

Reply:
good point, test added